.github/workflows/sims.yml

```diff
@@ -121,3 +121,32 @@ jobs:
         run: |
           make test-sim-multi-seed-short
         if: "env.GIT_DIFF != ''"
+
+  test-liveness:
+    runs-on: ubuntu-latest
+    needs: Build
+    timeout-minutes: 10
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2.1.3
+        with:
+          go-version: 1.15
+#      - uses: technote-space/get-diff-action@v4
```

Comment:
should this be commented?

Reply:
I believe so, the localnet action didn't execute when it was there. I believe it was there in order to cache the simulation binary, which doesn't seem to apply for the localnet sim. This is a good reminder though to re-enable the file filter before committing this PR.

Reply:
No, it shouldnt be. I had commented it just to trigger the action and test. Since it appears it's fine I will uncomment, commit, and merge the PR.
Thanks @marbar3778